recipe/0103-poetry-reading-annotations/annotations.json

```diff
@@ -0,0 +1,44 @@
+{
+  "@context": "http://iiif.io/api/presentation/3/context.json",
+  "id": "{{ id.url }}",
+  "type": "AnnotationPage",
+  "items": 
+  [
+   { 
+      "@context":"http://www.w3.org/ns/anno.jsonld",
```

Comment:
This `@context` should not be here.

Reply:
This is now resolved with: https://github.com/IIIF/cookbook-recipes/pull/130/commits/107ecabf6c30a7970bb03ebc42ccd12cb9fc1816